hydra-node/json-schemas/api.yaml

```diff
@@ -808,6 +812,16 @@ components:
           items:
             $ref: "#/components/schemas/Party"
 
+    HeadStatus:
+      type: string
+      enum:
+        - Idle
+        - Initializing
+        - Open
+        - Closed
+        - ClosedAfterDeadline
```

Comment:
@v0d1ch I'm sorry, but they actually wished `FanoutPossible` here. See #823 :)

Reply:
No worries, I'll change it quickly.

Reply:
...and done!